webpack.config.js

```diff
@@ -50,7 +50,6 @@ module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
   mode: env && env.NODE_ENV === 'development' ? 'development' : 'production',
   output: {
     filename: '[name].min.js',
-    libraryTarget: 'umd',
```

Comment:
Why are we removing this? What's the effect of this on the webex min and meetings min js

Reply:
We have added import from each module : webex, meeting and calling separately and in that we are already mentioning the type as umd. After that it doesn't seem that we need this library target anymore.
I did go to Meetings samples page and tested the webex initialization, was working fine hence assuming this change didn't have any impact on  webex min and meetings min js.